app/scripts/src/content/search.js

```diff
@@ -59,20 +59,24 @@ Search.prototype = {
   findEpisode: function(options) {
     this.findItem({
       success: function(response) {
-        Request.send({
-          method: 'GET',
-          url: this.getEpisodeUrl(response['show']['ids']['slug']),
-          success: function(resp) {
-            if (this.item.episode) {
-              options.success.call(this, JSON.parse(resp));
-            } else {
-              this.findEpisodeByTitle(resp, options);
+        if (!response) {
```

Comment:
It really happens if the show is not present on trakt.tv. (Or under a different title.) Right now I don't have an active netflix account so I can't give you any examples. But if it fails to match a show it will instead display an error in the console, Due to the response['show']['ids']['slug']. Whether it happens often or never, it is much better to log these either way.